### DISCUSSION TITLE
:b1 command is not work from some version?

### DISCUSSION THREAD

**Discussion by 839891627** [JetBrains/ideavim#386] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/10300651/138199870-a2088e03-70e0-4c18-8f2c-cc79aedc9570.png)

`:b1` is not work, but `:b#`  is still works

**Comment by lippfi** [comment, marked as answer]
Sorry for that, the problem is in our new Vimscript parser. It will take some time to make a better one, but we surely will fix that
Please use `:b 1` instead of `:b1` as a temporary solution

**Reply by 839891627** [reply]
oh, great! thx~ 😄

**Reply by lippfi** [reply]
The fix is available in the IdeaVim 1.8.0
